src/luascript.cpp

```diff
@@ -10463,6 +10541,52 @@ int LuaScriptInterface::luaHouseSetAccessList(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaHouseStartTrade(lua_State* L)
+{
+	// house:startTrade(player, tradePartner)
+	House* house = getUserdata<House>(L, 1);
+	Player* player = getUserdata<Player>(L, 2);
+	Player* tradePartner = getUserdata<Player>(L, 3);
+
+	if (!player || !tradePartner || !house) {
+		lua_pushnil(L);
+		return 1;
+	}
+
+	if (!Position::areInRange<2, 2, 0>(tradePartner->getPosition(), player->getPosition())) {
```

Comment:
In my opinion, that way is better to keep consistency of the method, avoiding the user to mess things up in case he tries to use it in other context.